packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -117,7 +123,7 @@ export interface NodegroupOptions {
    *
    * @default - None
    */
-  readonly labels?: { [name: string]: string };
+  readonly labels?: { [name: string]: string; };
```

Comment:
We usually omit this `;`:

```suggestion
  readonly labels?: { [name: string]: string };
```